products/gitlab.md

```diff
@@ -247,8 +255,10 @@ releases:
 > pipeline features, using an open-source license, developed by GitLab Inc.
 
 GitLab has a [well-defined versioning policy](https://docs.gitlab.com/ce/policy/maintenance.html)
-based on Semantic Versioning. New releases are announced on the [blog](https://about.gitlab.com/releases/categories/releases/),
-and you can subscribe to an [RSS Feed](https://about.gitlab.com/atom.xml) as well. A [tool is
+based on Semantic Versioning. New releases are made on [third Thursday of every
```

Comment:
The release cadence was documented in the table, so it must be updated there too.

Reply:
Missed that one, fixed now.